api/src/main/java/org/pmiops/workbench/db/dao/RdrExportDao.java

```diff
@@ -12,7 +12,7 @@
       nativeQuery = true,
```

Comment:
unrelated but we should try to avoid native queries wherever possible since our test db and production db are different.

Reply:
Since rdr_export doesn't have ENTITY_ID as a foreign key (since it can contain either WORKSPACE  or USER id) i was unable to use it the using Db classes.